uint32   WordId
float64  WordValue
